blockchain/pallets/minting/src/tests.rs

```diff
@@ -284,6 +284,8 @@ mod register_identity {
 
     #[cfg(test)]
     mod extension_proofs {
```

Comment:
This shouldn't be in the extension_proofs module. Probably goes in the root test module next to the other tests about minting registration.